tests/zeek/evt/import-from.zeek

```diff
@@ -47,8 +49,7 @@ public function y()  : string {
 
 # @TEST-START-FILE ssh.evt
 protocol analyzer spicy::SSH over TCP:
-    parse with SSH::Banner,
-    port 22/tcp;
+    parse with SSH::Banner;
```

Comment:
Could you add an additional test which ensures that a port in an evt file overrides whatever was already set in the parser?

Reply:
Same as above: this would register them multiple times, which is ok.